tests/polars/test_polars_check.py

```diff
@@ -243,7 +243,7 @@ def get_data_param(self):
                         self.sample_string_data, "binary"
                     ),
                 },
-                {"datatype": Categorical, "data": self.sample_string_data},
+                {"datatype": Categorical(ordering="physical"), "data": self.sample_string_data},
```

Comment:
@FilipAisot To fix the remaining test case we need to:
1. Remove the second pair of brackets on line 198, so change to: 
```python
"test_expression": ["a"],
```
2. Change line 158 to: 
```python
df = pl.LazyFrame(fail_case_data, orient="row", schema=polars_schema)
```
The second one fixes issues that come up with other Polars versions

Reply:
Fixed.